modal/queue.py

```diff
@@ -67,6 +69,21 @@ def __init__(self):
         obj = _Queue.new()
         self._init_from_other(obj)
 
+    @classmethod
+    @asynccontextmanager
+    async def ephemeral(cls: Type["_Queue"], client: Optional[_Client] = None, environment_name: Optional[str] = None):
```

Comment:
I *think* we can get correct type annotations/api autocomplete on the context variable in our type stubs if you use `@synchronicity.async_wrap.asynccontextmanager` instead (and annotate the return value as an `AsyncIterator["Queue"]`)